.github/actions/check-python-packaging/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Check Python Packaging"
+description: ""
+runs:
+  using: "composite"
+  steps:
+    - name: Install packaging dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools build wheel twine
```

Comment:
we probably need to set the lower bound of setuptools here as well

Reply:
good catch!